bind/entrypoint.sh

```diff
@@ -38,26 +34,16 @@ if [ -n "${_DAPPNODE_GLOBAL_DOMAIN}" ]; then
     domain=${_DAPPNODE_GLOBAL_DOMAIN}
     echo "Using existing domain: $domain"
 else
-    fetched_domain=$(fetch_env "DOMAIN")
-
-    if [ $? -eq 0 ]; then
-        domain=$fetched_domain
-    else
-        echo "[ERROR] Failed to fetch DOMAIN"
-    fi
+    fetched_domain=$(fetch_env "DOMAIN" | tail -n 1)
```

Comment:
Why not just use `env_var_value` variable? We don't need "hacky" solutions.

(Of course do not set it as local)